test/helper/error-from-worker.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const serializeError = require('../../lib/serialize-error');
+
+module.exports = function (err, {type, file, stack} = {}) {
```

Comment:
We're targeting Node.js 4 still, so this destructuring won't work unfortunately.

Perhaps just use `Object.assign()` without checking for values?

Reply:
Sure thing